gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -240,25 +280,22 @@ static public boolean intersectPolygonEdges (FloatArray polygon1, FloatArray pol
 		return false;
 	}
 
-	static Vector2 v2a = new Vector2();
-	static Vector2 v2b = new Vector2();
-	static Vector2 v2c = new Vector2();
-	static Vector2 v2d = new Vector2();
-
 	/** Returns the distance between the given line and point. Note the specified line is not a line segment. */
 	public static float distanceLinePoint (float startX, float startY, float endX, float endY, float pointX, float pointY) {
-		float normalLength = Vector2.len(endX - startX, endY - startY);
```

Comment:
What's wrong with `Vector2.len`?

Reply:
I didn't get the [this](https://github.com/libgdx/libgdx/commit/abb9f5b116a1470c7b23e0426a220b2515977165) commit. I'm reverting it